include/vcpkg/base/messages.h

```diff
@@ -1432,6 +1510,10 @@ namespace vcpkg
                     "Specify the target architecture triplet. See 'vcpkg help triplet'.\n(default: '{env_var}')");
     DECLARE_MESSAGE(StoredBinaryCache, (msg::path), "", "Stored binary cache: \"{path}\"");
     DECLARE_MESSAGE(StoreOptionMissingSha, (), "", "--store option is invalid without a sha512");
+    DECLARE_MESSAGE(SuccessfulyExported,
+                    (msg::package_name, msg::path),
+                    "",
+                    "Successfully exported {package_name}. Checkout {path}");
```

Comment:
@ras0219-msft Is this saying "checkout" as in "git checkout", or is it saying "check out" as in "look at"? If the latter I think it should say:

```
Exported {package_name} to {path}
```

or similar.

Reply:
Agree with Billy here, we should simplify the message.